packages/List/src/list-item.scss

```diff
@@ -290,6 +290,10 @@
         }
     }
 
+    .ids-visual-identifier__icon {
```

Comment:
I'd probably use the class: ids-list-item__visual since it's not just the icon that should have this, the avatar and color should too